content/en/docs/instrumentation/js/propagation.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Propagation
+description: Context propagation for the JS SDK
+aliases: [/docs/instrumentation/js/api/propagation]
+weight: 7
+---
+Propagation is the mechanism that moves data between services and processes.
+Although not limited to tracing, it is what allows traces to build causal
+information about a system across services that are arbitrarily distributed
+across process and network boundaries.
+
+## Context propagation with libraries
+
+For the vast majority of use cases, context propagation is done with
+instrumentation libraries.
+
+For example, if you have several Node.js services that communicate over HTTP,
+you can use the
+[express](https://www.npmjs.com/package/@opentelemetry/instrumentation-express)
+and [http](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)
+instrumentation libraries to automatically propagate trace context across
+services for you.
+
+**It is highly recommend that you use instrumentation libraries to progagate
+context.** Although it is possible to propagate context manually, if your system
+uses libraries to communicate between services, use a matching instrumentation
+library to propagate context.
+
+Refer to [Libraries](/docs/instrumentation/js/libraries) to learn more about
+instrumentation libraries and how to use them.
+
+## Manual W3C Trace Context Propagation
```

Comment:
It might be worth mentioning that official support for the W3C propagator is available (the `W3CTraceContextPropagator` from the package `@opentelemetry/core`) and this section demonstrates manual W3C propagation.